src/localize.js

```diff
@@ -51,6 +52,7 @@ var languages = {
   lt,
   ro,
   pt,
+  pt-br,
```

Comment:
This is an invalid name for a variable

Reply:
@denysdovhan any forecast to approve this PR? I would love to use it in my Home assistant.